lib/Doctrine/Common/Reflection/StaticReflectionParser.php

```diff
@@ -153,8 +153,10 @@ protected function parse()
                     $docComment = $token[1];
                     break;
                 case T_CLASS:
-                    $this->docComment['class'] = $docComment;
-                    $docComment = '';
+                    if ($docComment) {
```

Comment:
IMO this is more readable, because Intent here is to stop replacing docComment with blank value. When there is strict check, reader of the code searches for reasons why does it allow some blank values and there is none.